aries-site/src/pages/components/tooltip.mdx

```diff
@@ -0,0 +1,67 @@
+import { Example } from '../../layouts';
+import {
+  ToolTipDefinitionExample,  
+  ToolTipExitExample,
+  ToolTipIconExample,
+  ToolTipSimpleExample,
+} from '../../examples';
+
+<Example
+  code="https://raw.githubusercontent.com/grommet/hpe-design-system/master/aries-site/src/examples/components/tooltip/ToolTipSimpleExample.js"
+  docs="https://v2.grommet.io/tip?theme=hpe#props"
+  figma="https://www.figma.com/file/Jd34cI5yyZIPb5BIwkIPU2/HPE-Tooltip?node-id=0%3A1"
+>
+  <ToolTipSimpleExample />
+</Example>
+
+## Guidance
+
+Tooltips will display information upon hover and focus.
+Tooltips provides messages that give the user more information about a element.
+
+### Usage
+
+1. Don't use tooltips for information that is vital to a task completion.
+2. Provide a brief and helpful content inside the tooltip.
+3. Support both mouse and keyboard hover.
+4. Use tooltip arrows when multiple elements are nearby.
+5. Use tooltip consistently throughout your site.
+
+Make sure to test tooltip positioning to ensure that the content does not block other information pertient to the user's goal.
+
+### Accessibility
+
+Tooltips should support both mouse and keyboard hover.
+Do not use tooltips for information that is vital for the user to complete the task.
+
+## Variants
+
+There are a few different ways that you can use tooltip. 
+
+### Tooltip with icons
+
+An icon without a label should be provided a tooltip. Some of the icons have some level of ambiguity which is why this is recommended. This would be also a usecase for icons in any sidebar
```

Comment:
```suggestion
An icon without a label should be provided a tooltip. Some of the icons have some level of ambiguity which is why this is recommended. This would be also a use case for icons in any sidebar
```